Marlin/src/lcd/extui/lib/ftdi_eve_touch_ui/screens/bed_mesh_screen.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "../config.h"
 
-#if BOTH(TOUCH_UI_FTDI_EVE, AUTO_BED_LEVELING_UBL)
+#if BOTH(TOUCH_UI_FTDI_EVE, HAS_MESH)
```

Comment:
I already posted a PR which resolved this build issue in a different way: #20608.

I don't have compatible hardware, so I don't know which approach is correct. I would appreciate @InsanityAutomation's input on this.

Reply:
If it compiles with the other mesh-based leveling systems, then it is compatible. I've made a couple of other tweaks to that end.

Reply:
OK, I closed my other PR then.